ctapipe/core/logging.py

```diff
@@ -18,13 +18,8 @@ class ColoredFormatter(logging.Formatter):
 
     def format(self, record):
         """Format the LogRecord."""
-        rec = record.__dict__.copy()
-
-        rec["asctime"] = self.formatTime(record, self.datefmt)
-        rec["levelname"] = apply_colors(record.levelname)
-        rec["message"] = record.getMessage()
-
-        return self._fmt % rec
+        s = super().format(record)
+        return s.replace(record.levelname, apply_colors(record.levelname), 1)
```

Comment:
s is a string here right? is is safe to assume using replace will get the right one? If somebody changes the format string to also have the work INFO  or DEBUG in it somewhere, assuming it is the first one could be bad. (e.g. if you set the string to have the level be the last thing printed, but also have one of those words in the output)

Reply:
contrived example, if you had logs that looked like:
```
CTAPIPE-INFO Some output message [INFO]
```
where the part in brackets is the `levelname`, this would fail

Reply:
It would color the wrong info, yes.

Reply:
I am not sure if supporting this log format (as opposed to the default one we use) is worth copying the log record.

Reply:
alternately, just replace all instances. A bit of over-coloring is probably ok

Reply:
ok, removed the `1`, more colors!